pkg/sql/catalog/catprivilege/system.go

```diff
@@ -74,6 +74,8 @@ var (
 		catconstants.SpanStatsBuckets,
 		catconstants.SpanStatsSamples,
 		catconstants.SpanStatsTenantBoundaries,
+		catconstants.TxnExecInsightsTableName,
+		catconstants.StmntExecInsightsTableName,
```

Comment:
this is just a driveby comment, but the standard way we abbreviate this is `stmt`. I predict that using `stmnt` will cause a lot of confusion internally.